scripts/commands/getfame.lua

```diff
@@ -0,0 +1,73 @@
+-----------------------------------
+-- func: getfame
+-- desc: Gets fame level of a target player
+-----------------------------------
+local commandObj = {}
+
+commandObj.cmdprops =
+{
+    permission = 3,
+    parameters = 'si'
+}
+
+local function error(player, msg)
+    if msg == nil then
+        msg = '!getfame [player] <fame_zone 0-15>'
+    end
+
+    player:printToPlayer(msg)
+end
+
+commandObj.onTrigger = function(player, target, famezone)
+    -- validate target
+    local targ = nil
+
+    if target == nil then
+        if player:getCursorTarget() == nil then
+            targ = player
+        else
+            if player:getCursorTarget():isPC() then
+                targ = player:getCursorTarget()
+            else
+                error(player, 'You must target a player or specify a name.')
+                return
+            end
+        end
+    else
+        targ = GetPlayerByName(target)
+        if targ == nil then
+            error(player, string.format('Player named "%s" not found or not a valid player!', target))
+            return
+        end
+    end
+
+    local fameZoneNames = {}
+    for name, value in pairs(xi.quest.fame_area) do
```

Comment:
```suggestion
    for name, value in pairs(xi.fameArea) do
```

Reply:
Done, also fixed this in other areas where xi.quest.fame_area was still being used